pkg/logs/otel.go

```diff
@@ -18,6 +19,14 @@ func SetupOTelLogger(ctx context.Context, logger zerolog.Logger, config *types.O
 		return logger, nil
 	}
 
+	if os.Getenv("USER") == "" {
+		// If USER environment variable is not set, we are likely running in a container.
+		// Set it to "traefik" so that some libraries (like OpenTelemetry) do not fail to initialize.
```

Comment:
This could also happen in other environments. WDYT about:
```suggestion
        // If USER environment variable is not set, set it to "traefik" so that 
        // OpenTelemetry resource detection does not fail to initialize.
```